docs/FAQs/index.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: home
+title: FAQs
+nav_order: 6
+---
+
+# Estela FAQs  
+Check the list of common questions and issues in Estela, check the topics (e.g.**[Installation]**) and choose your question.
+
+## Estela Installation
+### Q: How to start again installation process if I had mistakes in the [*resources guide*]({% link estela/installation/resources.md %}){:target="_blank"}  
+**Ans:** If you had some mistakes, such as the ones listed in this section, during the installation process and want to start again you can use this command `$minikube delete --all --purge` to clean all the corrupted images generated and start again the process, this applies also If you have had troubles after [*variables guide*]({% link estela/installation/helm-variables.md %}){:target="_blank"}.
+
+### Q: How to solve the following error during `make resources` step:  
+```
+E1109 18:46:36.969158    6835 logs.go:192] command /bin/bash -c "sudo /var/lib/minikube/
+binaries/v1.25.3/kubectl describe nodes --kubeconfig=/var/lib/minikube/kubeconfig" failed
+ with error: /bin/bash -c "sudo /var/lib/minikube/binaries/v1.25.3/kubectl describe nodes
+  --kubeconfig=/var/lib/minikube/kubeconfig": Process exited with status 1
+stdout:
+stderr:
+The connection to the server localhost:8443 was refused - did you specify the right host or port?
+ output: "\n** stderr ** \nThe connection to the server localhost:8443 was refused - did you specify the right host or port?\n\n** /stderr **"
+❗  unable to fetch logs for: describe nodes
+```
+**Ans:** In some systems the following option `--feature-gates="TTLAfterFinished=true"`  on the `installation/Makefile` document gives an error, we will fix it soon, just delete the selected line.
```

Comment:
I think we should fix it instead of making a question.

Reply:
I have the same opinion but I don't know yet the purpose of that parameter so I don't wanna change it on the Makefile maybe in some systems work so I putted it aside as a question.